lib/segment/src/data_types/vectors.rs

```diff
@@ -115,7 +115,7 @@ impl TryFrom<Vector> for MultiDenseVector {
 
     fn try_from(value: Vector) -> Result<Self, Self::Error> {
         match value {
-            Vector::Dense(_) => Err(OperationError::WrongMulti),
+            Vector::Dense(v) => Ok(MultiDenseVector::new(v, 1)), // expand single dense vector
```

Comment:
this line is actually incorrect!
the second argument is the vector dimension and not the vector count :face_with_head_bandage:

Reply:
Cool that you noticed it )

Reply:
https://github.com/qdrant/qdrant/pull/4203/commits/0bfe07b6faed467c267306f39223961fe7afdc89